webrender/src/tiling.rs

```diff
@@ -1682,20 +1702,21 @@ pub struct StackingContext {
     composite_ops: CompositeOps,
     clip_source: ClipSource,
     clip_cache_info: Option<MaskCacheInfo>,
+    packed_layer_index: PackedLayerIndex,
 }
 
 #[derive(Debug, Clone)]
 #[repr(C)]
-pub struct PackedStackingContext {
+pub struct PackedLayer {
     transform: LayerToWorldTransform,
     inv_transform: WorldToLayerTransform,
     local_clip_rect: LayerRect,
     screen_vertices: [WorldPoint4D; 4],
 }
 
-impl Default for PackedStackingContext {
-    fn default() -> PackedStackingContext {
-        PackedStackingContext {
+impl Default for PackedLayer {
```

Comment:
Hrm. Okay. I'm happy to change it, but does that mean that the implementation of Default::default should change.